src/Knp/Snappy/Pdf.php

```diff
@@ -52,49 +51,12 @@ protected function handleOptions(array $options = array())
      */
     public function generate($input, $output, array $options = array(), $overwrite = false)
     {
-        $options = $this->handleOptions($options);
+        $options = $this->handleOptions(array_merge($this->getOptions(), $options));
```

Comment:
@akovalyov do you think we should use this `$options = $this->mergeOptions($options);` here, or not?